### ISSUE TITLE
v1.34 - virus warning

### ISSUE THREAD

**Issue by somera** [irusanov/ZenTimings#58]
Hi,

when I try to download the current version I get this

![image](https://github.com/user-attachments/assets/83df8a00-2f9d-45b7-959c-fdaac80a0162)

in Chrome.

And https://www.virustotal.com/gui/url/cbcd9b649a5c5f898ccc2e1216ba0731b66487994ad969f0f1e20f1e31571384

![image](https://github.com/user-attachments/assets/83a8e316-021c-4472-8762-8e85aac39a84)

**Comment by irusanov** [comment]
I think that is some strange problem with the archiving method. I usually do it manually with 7zip, but used a programmatic creation of the archive this time. I've zipped the same files manually and Defender does not complain anymore. Don't think there's a real problem, but I will replace the zip with the one I've manually zipped. Need to update the checksum though.

**Comment by irusanov** [comment]
Thanks for letting me know. I wanted to automate the process a bit as there were too many manual operations to prepare a release.

I've compared the files in the bad archive and the ones I released from and manually zipped now and they are identical, so I'm confident it is false-positive. New zip works.

**Comment by irusanov** [comment]
Some more details:

If I use the program to zip the files, but remove just the "ZenTimings.exe.config", then the resulting zip comes clean. If I zip just the config file and leave everything else, then the zip is also clean.

It's some sort of a heuristic pattern that is hit, I guess. Will use a manual zip for the next releases or just the simple
```csharp
ZipFile.CreateFromDirectory(releaseDir, zipReleasePath, CompressionLevel.SmallestSize, false);
```

The config file is nothing special, only contains the target framework version
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
<configSections>
    <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
        <section name="ZenTimings.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false"/>
    </sectionGroup>
</configSections>
<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/></startup>
</configuration>

```

**Comment by somera** [comment]
I assumed it was a false flag. But you never know.

**Comment by irusanov** [comment]
Well, to be fair, I panicked a bit that something might have went wrong on my end. It's the last thing I would want, especially when there's an auto update. That would be a disaster...

**Comment by somera** [comment]
I don't get any virus warning in 1.35.

**Comment by irusanov** [comment]
Yes, I'm sticking to manual zipping the files for now, which fixes the issue. I've also made sure to check not only with VirusTotal, but with Defender as well, which was detecting it before.
